services/docker-compose.yml

```diff
@@ -1134,7 +1134,10 @@ services:
       POSTGRES_DB: ${POSTGRES_DB}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
       POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_READONLY_PASSWORD: ${POSTGRES_READONLY_PASSWORD}
+      POSTGRES_READONLY_USER: ${POSTGRES_READONLY_USER}
     volumes:
+      - ./postgres/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
```

Comment:
I am not sure if this will work out. We don't keep the files on the filesystem (there is no simcore code on running machines). 

Another problem here is: it forces the container to the node where this file exists. This affects H/A and Restartability. Potential solution is to use docker configs :bulb:

Reply:
I agree, this will not work sadly